http/src/main/java/io/hyperfoil/http/steps/HttpResponseHandlersImpl.java

```diff
@@ -286,7 +290,10 @@ public void handleEnd(HttpRequest request, boolean executed) {
                      handler.after(request.session);
                   }
                }
-               HttpCache.get(request.session).tryStore(request);
+               HttpCache httpCache = HttpCache.get(request.session);
```

Comment:
it would be great to not query it if we know there should be none (by checking the cache control first?)

Reply:
Done :white_check_mark: